bin/tlint

```diff
@@ -21,14 +21,10 @@ if (file_exists($appAutoloader = getcwd() . '/vendor/autoload.php')) {
 }
 
 use Symfony\Component\Console\Application;
-use Tighten\Commands\FormatCommand;
-use Tighten\Commands\LintCommand;
-use function Tighten\version;
-
-$application = new Application(
-        'tlint',
-        version()
-);
+use Tighten\TLint\Commands\FormatCommand;
+use Tighten\TLint\Commands\LintCommand;
+
+$application = new Application('tlint', 'v5.0.16');
```

Comment:
This should probably be v6.0.0 since it requires a bump, right?

Reply:
Also, could we maybe make this as a constant at the top, so it's A) globally accessible in case we want it, but more importantly B) really easy to find and bump? maybe `TLINT_VERSION` or something

Reply:
Yep sounds good, that also makes replacing it programatically a bit easier, which I couldn't quite figure out this morning but now it works! Added a script for that too, it's not the prettiest but it works.

Reply:
Very clean! Thank you @bakerkretzmar ❤️